bindings/go/src/fdb/snapshot.go

```diff
@@ -86,3 +86,11 @@ func (s Snapshot) GetReadVersion() FutureInt64 {
 func (s Snapshot) GetDatabase() Database {
 	return s.transaction.db
 }
+
+func (s Snapshot) GetEstimatedRangeSizeBytes(r ExactRange) FutureInt64 {
+	begin, end := r.FDBRangeKeySelectors()
```

Comment:
`ExactRange` provides a more straightforward way to access the keys in `r.FDBRangeKeys()`, which probably makes this and the other version of this function a little bit clearer.

Reply:
Done.